client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react"
+import { Grid, Container, Typography, Link } from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import useStyles from './useStyles';
+import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
+import EditProfile from '../../components/EditProfile/EditProfile'
+import ProfilePhoto from '../../components/ProfilePhoto/ProfilePhoto'
+import Payment from '../../components/Payment/Payment'
+import Security from '../../components/Security/Security'
+import Settings from '../../components/Settings/Settings'
+
+
+export default function Profile(): JSX.Element {
+  const classes = useStyles();
+
+  const [currentSection, setCurrentSection] = useState("EditProfile")
+
+  const handleClick = (section: string) => {
+    setCurrentSection(section)
+  }
+
+  return (
+    <Grid container className={`${classes.root}`}>
+      <CssBaseline />
+      {/* <ChatSideBanner /> */}
+      <Grid className={`${classes.menuItems}`}>
+        <Link 
+          onClick={() => handleClick("EditProfile")} 
+          className={`${classes.menuItem} ${currentSection === "EditProfile" && classes.selectedMenuItem}`} 
+          underline="none"
+        > 
+          Edit Profile 
+        </Link>
```

Comment:
I would in the future think of a way this can be refactored to be more dynamic and reduce down this file.

Reply:
**Note** You don't need to worry about this in this PR but in the future if you have time consider a refactor on this component.